src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -168,10 +168,10 @@
     (catch Exception e
       (log/error e (trs "Error while sweeping reports")))))
 
-(defn gc-packages! [ttl db]
+(defn gc-packages! [db]
```

Comment:
It looks like it was an arity problem. I figure it wasn't ever seen because the ```when``` that calls this function didn't trigger.